.travis.yml

```diff
@@ -10,6 +10,6 @@ before_install:
   - if ! go get github.com/golang/tools/cmd/cover; then go get golang.org/x/tools/cmd/cover; fi
 script:
   - go test ./...
-  - go test -c -covermode=count -coverpkg=github.com/cweill/gotests,github.com/cweill/gotests/internal/input,github.com/cweill/gotests/internal/render,github.com/cweill/gotests/internal/goparser,github.com/cweill/gotests/internal/output,github.com/cweill/gotests/internal/models
+  - go test -c -covermode=count -coverpkg=github.com/zaynjarvis/gotests,github.com/zaynjarvis/gotests/internal/input,github.com/zaynjarvis/gotests/internal/render,github.com/zaynjarvis/gotests/internal/goparser,github.com/zaynjarvis/gotests/internal/output,github.com/zaynjarvis/gotests/internal/models
```

Comment:
Please replace `zaynjarvis` with `cweill` throughout this PR.

Reply:
Oh sorry, that must be the changes I made to test the module after this PR. Just updated.